log.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"os"
+	"regexp"
+)
+
+const (
+	// Severity.
+
+	// From /usr/include/sys/syslog.h.
+	// These are the same on Linux, BSD, and OS X.
+	LOG_EMERG int = iota
+	LOG_ALERT
+	LOG_CRIT
+	LOG_ERR
+	LOG_WARNING
+	LOG_NOTICE
+	LOG_INFO
+	LOG_DEBUG
+)
+
+const (
+	EnvLogLevel      = "LOG_LEVEL"
+	EnvLogFormat     = "LOG_FORMAT"
+	defaultLogFormat = "<%s>\t%s" // #1 severity, #2 msg
+)
+
+type Logger interface {
+	Log(severity int, message string)
+}
+
+var logs Logger
+var logLevel = LOG_INFO
+var logFormat = defaultLogFormat
+
+func init() {
+	if lvl, ok := os.LookupEnv(EnvLogLevel); ok {
+		switch lvl {
+		case "DEBUG":
+			logLevel = LOG_DEBUG
+		case "ERROR":
+			logLevel = LOG_ERR
+		default:
+			logLevel = LOG_INFO
+		}
+	}
```

Comment:
Actually I don't think we need this configurable at all. Or at least we need `DEBUG` default both for stdout+err and syslog.